src/mavedb/routers/experiments.py

```diff
@@ -125,7 +125,7 @@ def get_experiment_score_sets(
     #
     # TODO(#182): A side effect of this implementation is that only the user who has created the experiment may view all the Score sets
     # associated with a given experiment. This could be solved with user impersonation for certain user roles.
-    score_sets = db.query(ScoreSet).filter(ScoreSet.experiment_id == experiment.id)
+    score_sets = db.query(ScoreSet).filter(ScoreSet.experiment_id == experiment.id).filter(~ScoreSet.superseding_score_set.has())
```

Comment:
Tried this out by looking at https://staging.mavedb.org/experiments/urn:mavedb:00000674-a but staging is still on 2024.2.0 according to https://api.staging.mavedb.org/api/v1/api/version

Reply:
@afrubin The problem is both urn:mavedb:00000674-a-3 and urn:mavedb:00000674-a-4 replace urn:mavedb:00000674-a-1. It leads to the codes being confused. It should be only one superseded score set or do we allow multiple score sets replacing the same score set? I think we need to modify the data in database.

Reply:
But other superseded score sets also get problems in the experiment score set list. I'll keep digging it.